### PR TITLE
Add security document id in getCredentialsSNMPv1/getCredentialsSNMPv3

### PR DESCRIPTION
**Pull request by ericclappier-eaton**

### CODE REVIEW

File: src/fty_common_nut_credentials.cc [hunk 1]
```diff
@@ -63,6 +63,8 @@ std::vector<CredentialsSNMPv3> getCredentialsSNMPv3(DocumentFilter pred)
 
             auto cred = dynamic_cast<const secw::Snmpv3*>(i.get());
             if (cred) {
+                const secw::Document* doc = i.get();
+                secw::Id doc_id = doc->getId();
```
Comment (boricj) [inline src/fty_common_nut_credentials.cc:67]: Keep same code convention with rest of source: `docId`.

File: include/fty_common_nut_credentials.h [hunk 2]
```diff
@@ -29,24 +29,27 @@
...
-    CredentialsSNMPv3(const std::string& name,
+    CredentialsSNMPv3(const secw::Id id, const std::string& name,
         const std::string& authPass, const std::string& authProto,
         const std::string& privPass, const std::string& privProto
-    ) : secName(name),
+    ) : document_id(id), secName(name),
         authPassword(authPass), authProtocol(authProto),
         privPassword(privPass), privProtocol(privProto) {}
 
+    secw::Id document_id;
```
Comment (boricj) [inline include/fty_common_nut_credentials.h]: Keep same code convention with rest of structure: `documentId`.

File: include/fty_common_nut_credentials.h [hunk 3]
```diff
@@ -29,24 +29,27 @@
...
+#include "secw_document.h"
+
 namespace nutcommon {
 
 struct CredentialsSNMPv1
 {
-    CredentialsSNMPv1(const std::string& comm) : community(comm) {}
-
+    CredentialsSNMPv1(const secw::Id id, const std::string& comm) : document_id(id), community(comm) {}
+    secw::Id document_id;
```
Comment (boricj) [inline include/fty_common_nut_credentials.h]: Keep same code convention with rest of source: documentId.

**Review by boricj — CHANGES_REQUESTED** [review]
Minor code styling changes required.

**Review by perrettecl — APPROVED** [review]

File: include/fty_common_nut_credentials.h (hunk 3 above)
Comment (ericclappier-eaton) [reply include/fty_common_nut_credentials.h]: Thanks take into account

File: include/fty_common_nut_credentials.h (hunk 2 above)
Comment (ericclappier-eaton) [reply include/fty_common_nut_credentials.h]: Thanks take into account

**Review by boricj — APPROVED** [review]

**Pull request merged**